web/admin/src/ContentEditor.js

```diff
@@ -25,48 +25,49 @@ import ContentPreview from './ContentPreview'
 export default class ContentEditor extends PureComponent {
   constructor(props) {
     super()
+    this.state = {
+      isEditing: false,
+    }
+  }
+
+  onUpdate = (contentItem, prop, value) => {
+    this.props.onUpdate(contentItem, prop, value)
+    this.setState({ isEditing: true })
+    setTimeout(() => this.setState({ isEditing: false }), 1000)
```

Comment:
Why set state to isEditing: true for one second, then set to false?  Add comments explaining what this is intending, since this seems like a workaround for something.

Reply:
Thats correct. Josh doesn't think its clear that updates are autosaving so we need a way of representing that in some capacity thus showing the text "Saving..." for one second upon making a change

Reply:
👍 let's rename to `showSaving` or something more indicative of what this state variable signifies.